apps/src/applab/ExternalRedirectDialog.jsx

```diff
@@ -7,11 +7,21 @@ import i18n from '@cdo/locale';
 import {connect} from 'react-redux';
 import {actions, REDIRECT_RESPONSE} from './redux/applab';
 import {studio} from '@cdo/apps/lib/util/urlHelpers';
+import {BASE_DIALOG_WIDTH} from '@cdo/apps/constants';
 
 const styles = {
   title: {
     display: 'inline',
     wordWrap: 'break-word'
+  },
+  url: {
+    display: '-webkit-box',
+    WebkitLineClamp: 3,
+    WebkitBoxOrient: 'vertical',
+    overflow: 'hidden',
+    maxWidth: '100%',
+    wordWrap: 'break-word',
+    maxHeight: '140px'
```

Comment:
Note: MaxHeight is set so long urls in IE11 don't cause the dialog to expand beyond the bottom of the page. The webkit settings here aren't honored in IE11.